clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/config/KubernetesConfigurationProperties.java

```diff
@@ -75,4 +77,21 @@ public void validate() {
       }
     }
   }
+
+  public KubernetesConfigurationProperties() {}
+
+  /**
+   * For larger number of Kubernetes accounts, SpringBoot based properties binding is taking tens of
+   * minutes, hence this custom constructor for KubernetesConfigurationProperties is needed.
+   *
+   * @param kubernetesConfigurationProvider This provides the KubernetesConfigurationProperties
+   *     object
+   */
+  public KubernetesConfigurationProperties(
```

Comment:
I'm not sure I understand this constructor...it looks like it is never setting the accounts so the resulting properties will always have an empty list of accounts?

In general, I'm not sure why we need this constructor, which creates another `KubernetesConfigurationProperties` inside the constructor.  Couldn't we just replace every call to `KubernetesConfigurationProperties(kubernetesConfigurationProvider)` with kubernetesConfigurationProvider. getKubernetesConfigurationProperties() and avoid the extra call?

Reply:
Agreed and removed the constructor from KubernetesConfigurationProperties.